api/src/main/proto/stellarstation/api/v1/radio/radio.proto

```diff
@@ -60,6 +63,26 @@ message CommunicationProtocol {
   }
 }
 
+// The type of polarization the antenna will use.
+//
+// https://en.wikipedia.org/wiki/Antenna_(radio)#Polarization
+enum Polarization {
+  // Any polarization can be used for communications.
+  ANY = 0;
+
+  // Horizontal polarization.
+  H = 1;
```

Comment:
`V` and `H` are too short to be meaningful names in code that uses it (and imagine if someone ended up static importing them, ugh...). Let's go with HORIZONTAL and VERTICAL for them, not that long.